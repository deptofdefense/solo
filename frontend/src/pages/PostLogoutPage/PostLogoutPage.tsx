import React from "react";
import classes from "./PostLogoutPage.module.css";

const PostLoginPage: React.FC = () => (
  <div className={classes.root}>
    <h2>You have been logged out.</h2>
    <div>Close your browswer window</div>
  </div>
);

export default PostLoginPage;
