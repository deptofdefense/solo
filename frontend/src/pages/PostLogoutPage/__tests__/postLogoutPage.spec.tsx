import React from "react";
import PostLogoutPage from "../PostLogoutPage";
import { render } from "test-utils";

describe("post logout page componenent", () => {
  it("matches snapshot", async () => {
    const { asFragment } = render(<PostLogoutPage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
