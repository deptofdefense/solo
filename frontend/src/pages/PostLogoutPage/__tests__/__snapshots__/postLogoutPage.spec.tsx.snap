// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`post logout page componenent matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="root"
  >
    <h2>
      You have been logged out.
    </h2>
    <div>
      Close your browswer window
    </div>
  </div>
</DocumentFragment>
`;
